### PR TITLE
[WOE] Implement Blossoming Tortoise

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Blossoming Tortoise]]

**Comment by Susucre** [comment]
```Error: (subtypes) [Turtle] != [] for WOE - Blossoming Tortoise - 163
Error: (types) [Creature] != [Creature, Turtle] for WOE - Blossoming Tortoise - 163```

Well, gonna add the turtoise to the WOE exception.

File: Mage.Sets/src/mage/cards/b/BlossomingTortoise.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ *
+ * @author Susucr
+ */
+public final class BlossomingTortoise extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("land creatures you control");
+
+    static {
+        filter.add(CardType.LAND.getPredicate());
+        filter.add(TargetController.YOU.getControllerPredicate());
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/b/BlossomingTortoise.java:37]: you don't need this predicate or "you control" in the message if you're using `BoostControlledEffect`

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/b/BlossomingTortoise.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+import java.util.UUID;
+
+/**
+ *
+ * @author Susucr
+ */
+public final class BlossomingTortoise extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("land creatures you control");
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/b/BlossomingTortoise.java:33]: you can also get rid of "you control" here for the same reason

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/b/BlossomingTortoise.java (hunk 2 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/b/BlossomingTortoise.java:33]: oh right, I'll definitively try without adding those text before testing the cards in the future.

**Pull request merged**